app/services/testdata/InProgressQuestionnaireStatusGenerator.scala

```diff
@@ -42,61 +42,61 @@ trait InProgressQuestionnaireStatusGenerator extends ConstructiveGenerator {
     val didYouLiveInUkBetween14and18Answer = Random.yesNo
     def getWhatWasYourHomePostCodeWhenYouWere14 = {
       if (didYouLiveInUkBetween14and18Answer == "Yes") {
-        Some(PersistedQuestion("What was your home postcode when you were 14?",
-          PersistedAnswer(Some(Random.homePostcode), None, None)))
+        Some(QuestionnaireQuestion("What was your home postcode when you were 14?",
+          QuestionnaireAnswer(Some(Random.homePostcode), None, None)))
       } else {
         None
       }
     }
 
     def getSchoolName14to16Answer = {
       if (didYouLiveInUkBetween14and18Answer == "Yes") {
-        Some(PersistedQuestion("Aged 14 to 16 what was the name of your school?",
-          PersistedAnswer(Some(Random.age14to16School), None, None)))
+        Some(QuestionnaireQuestion("Aged 14 to 16 what was the name of your school?",
+          QuestionnaireAnswer(Some(Random.age14to16School), None, None)))
       } else {
         None
       }
     }
 
     def getSchoolName16to18Answer = {
       if (didYouLiveInUkBetween14and18Answer == "Yes") {
-        Some(PersistedQuestion("Aged 16 to 18 what was the name of your school?",
-          PersistedAnswer(Some(Random.age16to18School), None, None)))
+        Some(QuestionnaireQuestion("Aged 16 to 18 what was the name of your school?",
+          QuestionnaireAnswer(Some(Random.age16to18School), None, None)))
       } else {
         None
       }
     }
 
     def getFreeSchoolMealsAnswer = {
       if (didYouLiveInUkBetween14and18Answer == "Yes") {
-        Some(PersistedQuestion("Were you at any time eligible for free school meals?",
-          PersistedAnswer(Some(Random.yesNoPreferNotToSay), None, None)))
+        Some(QuestionnaireQuestion("Were you at any time eligible for free school meals?",
+          QuestionnaireAnswer(Some(Random.yesNoPreferNotToSay), None, None)))
       } else {
         None
       }
     }
 
     def getHaveDegreeAnswer = {
       if (generatorConfig.isCivilServant) {
-        Some(PersistedQuestion("Do you have a degree?",
-          PersistedAnswer(Some(if (generatorConfig.hasDegree) { "Yes" } else { "No" }), None, None))
+        Some(QuestionnaireQuestion("Do you have a degree?",
+          QuestionnaireAnswer(Some(if (generatorConfig.hasDegree) { "Yes" } else { "No" }), None, None))
```

Comment:
Definitely a question for the wider team, but - why are we not using booleans here?